core/src/main/java/jenkins/model/GlobalConfigurationCategory.java

```diff
@@ -73,11 +71,12 @@ public String getDisplayName() {
     public static class Security extends GlobalConfigurationCategory {
         @Override
         public String getShortDescription() {
-            return Messages.GlobalSecurityConfiguration_Description();
+            return hudson.security.Messages.GlobalSecurityConfiguration_Description();
```

Comment:
BTW not an issue introduced in this PR, but would be nice at some point to use `core/move-l10n.groovy` to put this message in the natural place.